website/src/views/NotFound/NotFound.test.jsx

```diff
@@ -11,4 +18,15 @@ describe('NotFound View', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot(); // eslint-disable-line no-undef
   });
+
+  /* enzyme testing */
+  const wrapper = shallow(<NotFound />);
+
+  it('should contains a sub Title to say what is going on', () => {
+    expect(wrapper.contains(<h2 className="subTitle">404 Page Not Found</h2>)).toBe(true);
+  });
+
+  it('should contains a button to go back', () => {
+    expect(wrapper.exists('.home-button')).toBe(true);
```

Comment:
Test name is a little inaccurate this is checking if there's a home button css class right?